### PR TITLE
Consensus::organize module consensus.

### PR DESCRIPTION
**Pull request by aucusaga**

### CODE REVIEW

**Review by chunhui01 — APPROVED** [review]

File: kernel/consensus/context/context.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+// 特定共识的字段标示
+type ConsensusConfig struct {
+	// 获取本次共识的类型名称
+	ConsensusName string `json:"consensusName,omitempty"`
+	// 获取本次共识的起始高度
+	BeginHeight int64 `json:"beginHeight,omitempty"`
+	// 获取本次共识的起始Blockid，即起始高度的上一区块blockid
+	BeginBlockid []byte
```
Comment (qizheng09) [inline kernel/consensus/context/context.go:35]: BeginBlockid and BeginHeight may repeated.

File: kernel/consensus/pluggable_consensus.go [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+func (sc *StepConsensus) put(con ConsensusInterface) error {
+	sc.mutex.Lock()
+	defer sc.mutex.Unlock()
+	sc.cons = append(sc.cons, con)
+	sc.stepIndex++
+	return nil
+}
+
+// 获取最新的共识实例
+func (sc *StepConsensus) get() ConsensusInterface {
```
Comment (qizheng09) [inline kernel/consensus/pluggable_consensus.go:40]: Get usually with a param. Can this renamed as `pop` or `getLatestCons`？

File: kernel/consensus/pluggable_consensus.go [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+	//getCurrentConsensusComponent
+	sc.mutex.RLock()
+	sc.mutex.RUnlock()
+	if len(sc.cons) == 0 {
+		return nil
+	}
+	return sc.cons[sc.stepIndex-1]
```
Comment (qizheng09) [inline kernel/consensus/pluggable_consensus.go:47]: Why not use the length of StepConsensus?

File: kernel/consensus/config/log.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# 模块名称
+module: xchain
```
Comment (qizheng09) [inline kernel/consensus/config/log.yaml:2]: Why the log config in this package.

**Pull request merged**